src/index.html

```diff
@@ -2,34 +2,19 @@
 <html>
 	<head>
 		<title>Angular 2 Grid Demo</title>
-		<link rel="stylesheet" href="bower_components/bootstrap/dist/css/bootstrap.min.css" />
-		<script src="node_modules/es6-shim/es6-shim.min.js"></script>
-		<script src="node_modules/zone.js/dist/zone.js"></script>
-		<script src="node_modules/reflect-metadata/Reflect.js"></script>
-		<script src="node_modules/systemjs/dist/system.js"></script>
+		<link rel="stylesheet" href="/node_modules/bootstrap/dist/css/bootstrap.min.css" />
+		<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
```

Comment:
The original index.html was calling in bower_components/bootstrap which was probably copied over from another source tree. I was pulling in the bootstrap library for the css which was already in the index and instead of including the whole jquery library just referenced it from google directly.